tools/docker/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
firecracker-containerd needs entrypoint.sh to run firecracker-containerd as a background process and have "go test" as a foreground process. But SDK doesn't have to follow this pattern.

Reply:
The reason why I did this is the `cp` was failing when I was doing it all on one command. So I decided to throw it in an entrypoint script and everything ended up working there.

Reply:
What was the error you got?

Reply:
Basically, it couldn't stat the file, firecracker and jailer, even though there were there o.o.

Reply:
That's odd. I'm fine merging this PR but may try to remove the entrypoint.sh later.